.github/workflows/ci.yml

```diff
@@ -427,6 +427,22 @@ jobs:
           cd $RUNNER_TEMP
           npm install $RUNNER_WORKSPACE/developer-tooling/packages/sdk/base/*.tgz
           npm install $RUNNER_WORKSPACE/developer-tooling/packages/sdk/utils/*.tgz
+  pack-packages:
+    name: Build Package tarballs for dry run
+    runs-on: ['self-hosted', 'org', '8-cpu']
+    needs: install-dependencies
+    steps:
+      - uses: actions/checkout@v4
+      - name: Sync workspace
+        uses: ./.github/actions/sync-workspace
+        with:
+          artifacts_to_cache: ${{ needs.install-dependencies.outputs.artifacts_to_cache }}
+      - name: Pack all packages into tarballs
+        run: |
+          pwd
+          mkdir -p ${{ github.workspace }}/artifacts
+          yarn workspaces foreach --all run make --out ${{ github.workspace }}/artifacts/%s-%v.tgz
```

Comment:
what happens later to those packages?

Reply:
nothing. actually when i started this it was before you added the pack and installing pkg so maybe ill remove